src/__test__/reducers/rolesReducer.test.js

```diff
@@ -100,7 +87,7 @@ it("should accept the deletion of permission", () => {
     const state = [
         {
             id: 1,
-            name: "Admin",
+            rolename: "Admin",
```

Comment:
why are you changin the property from role to rolename?

Reply:
because that's the name of a field in db